app/assets/stylesheets/fae/modules/forms/_radio.scss

```diff
@@ -7,7 +7,7 @@
 }
 
 .radio_collection--vertical {
-  label {
+  .radio label {
```

Comment:
I just clicked through to the CSS lint and don't agree with it. Different explanation: the code style through the app doesn't nest selectors on one line. This was done to a) visually discourage deep nesting and thereby reduce complex selectors and b) make property additions easier to track in Git because everything occupies its own line. 

```
.radio {
  label {
```